.travis.yml

```diff
@@ -14,14 +14,14 @@ before_install:
     - conda update --all --yes
     - travis_retry conda create --yes -n TEST $CONDA --file requirements.txt
     - source activate TEST
-    - travis_retry conda install --yes --file requirements-dev.txt
+    - travis_retry pip install -r requirements-dev.txt
```

Comment:
Note to self:  `pytest-cov` is unavailable in the default channel and outdated in the IOOS channel. For now it is OK to install the light weight dependencies (`requirements-dev.txt`) with `pip` and the heavy weight (`netCDF4-python`, `scipy`, etc) with `conda`.

The long term goal is to add those packages to conda-forge and just add that channel here.